app/models/sfo_case.rb

```diff
@@ -1,8 +1,10 @@
 class SfoCase < Document
   validates :sfo_case_state, presence: true
+  validates :sfo_case_opened_date, allow_blank: true, date: true
```

Comment:
Can you please check with the users if they want to have this as a required field or not? It was not clear to me that it's optional from the Zendesk ticket.

Reply:
Have checked and they want it to be mandatory, have amended the validation